docs/cypress-testing-library/intro.md

```diff
@@ -27,9 +27,8 @@ and `queryAllBy` commands off the global `cy` object.
 [See the `DOM Testing Library` docs for reference](https://testing-library.com/docs/dom-testing-library/api-queries).
 
 > Note: the `get*` queries are not supported because for reasonable Cypress tests you
-> need retryability and `find*` queries already support that. The reason the `query*`
-> variants are supported is to allow you to assert that elements do _not_ appear on
-> the screen.
+> need retryability and `find*` queries already support that. `query*` queries are not
```

Comment:
`query*` queries are no longer needed for asserting that elements do not appear in the DOM. A `find*` query can do that instead:

```js
// Doesn't exist now
cy.findByText('Non-existing Button Text').should('not.exist')

// Eventually won't exist
cy.findByText('Eventually will not exist').should('not.exist')
```

Reply:
I'd include a note about why they are "not necessary", including version when the find* change was added, and mention plans for future deprecation here.

Reply:
Agreed with @alexkrolick here 👍

Reply:
I think `find` was changed in https://github.com/testing-library/cypress-testing-library/pull/78 and released as `5.0.0`

Reply:
How does the following sound?

https://github.com/testing-library/testing-library-docs/pull/373/files#diff-6245e2573aa625d927375e2065f25b1cR29-R32

> Note: the `get*` queries are not supported because for reasonable Cypress tests you
> need retryability and `find*` queries already support that. `query*` queries are no longer
> necessary since v5 and will be removed in v6. `find*` fully support built-in Cypress
> assertions (removes the only use-case for `query*`).